Testscripts/Linux/run_dpdk_ring_latency.sh

```diff
@@ -8,6 +8,7 @@ HOMEDIR=$(pwd)
 export RTE_SDK="${HOMEDIR}/dpdk"
 export RTE_TARGET="x86_64-native-linuxapp-gcc"
 export DPDK_RING_PING_PATH="${HOMEDIR}/dpdk-ring-ping"
+export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/lib64/pkgconfig/"
```

Comment:
Why this change is required?

Reply:
this is actually a change that matthew and I had made earlier. It is a fix to a dpdk test

Reply:
The problem that prompted this change was that pkgconfig on RHEL 7 was not checking /usr/local/lib64/pkgconfig/ path
The fix for this problem was to add /usr/local/lib64/pkgconfig/ to PKG_CONFIG_PATH

Reply:
@johnsongeorge-w I can remove the commit with the "export" change if you would like so that only my changes to the NVME test case are merged.